packages/bruno-electron/src/ipc/network/oauth2-helper.js

```diff
@@ -29,9 +29,11 @@ const resolveOAuth2AuthorizationCodeAccessToken = async (request, collectionUid)
     code: authorizationCode,
     redirect_uri: callbackUrl,
     client_id: clientId,
-    client_secret: clientSecret,
-    state: state
+    client_secret: clientSecret
   };
+  if (state) {
+    data.state = state;
```

Comment:
Actually, I believe the state parameter should not be sent to access_token endpoint at all - it may be required by some services when directing user browser to authorization url, and at this stage of the flow bruno should at most verify if the state parameter was returned from `getOAuth2AuthorizationCode` - or maybe ignore it, because xsrf attacks the state parameter protects from are not a threat in Bruno integrated browser (may change if  #2650 is implemented though).

As a short term solution, I suggest removing line 33, and skip the `if` part entirely.

Reply:
You might be right - updated!